async-race/src/components/Race.vue

```diff
@@ -0,0 +1,154 @@
+<script setup lang="ts">
+import { useGarageStore } from "../store/garageStore.ts";
+import { useWinnersStore } from "../store/winnersStorage.ts";
+import { storeToRefs } from "pinia";
+import type { Car } from "../common/model.ts";
+import { ref, watch } from "vue";
+
+interface AnimationOptions {
+  animation: string;
+  animationPlayState: string;
+  backgroundColor?: string;
+}
+
+const props = defineProps<{ car: Car }>();
+
```

Comment:
Можно просто 
```suggestion
const props = defineProps<Car>();
```

и потом выше v-bind="car"